cosmos/runtime/txpool/ante.go

```diff
@@ -43,9 +42,12 @@ func (m *Mempool) AnteHandle(
 	msgs := tx.GetMsgs()
 
 	// We only want to eject transactions from comet on recheck.
-	if ctx.ExecMode() == sdk.ExecModeReCheck {
+	if ctx.ExecMode() == sdk.ExecModeCheck || ctx.ExecMode() == sdk.ExecModeReCheck {
```

Comment:
q: are there any metrics graphing the size of the tx-pool? Would be interested to see whether there is inconsistency / how large is it?